client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -470,7 +494,11 @@ function CompatCell({
             : undefined
         }
       >
-        <CellText {...{ support }} browser={browserInfo} />
+        <CellText
+          {...{ support }}
+          browser={browserInfo}
+          use_note_symbol={true}
+        />
         <CellIcons support={support} />
```

Comment:
I believe the changes above are not necessary, if we just add the footnote icon to the `CellIcons` here.